### PR TITLE
Upgrade to 2.15

### PR DESCRIPTION
**Pull request by williamkbentley**

### CODE REVIEW

File: src/backend/main.ts [hunk 1]
```diff
@@ -63,9 +63,8 @@ const initialize = async () => {
       developmentServer: process.env.NODE_ENV === "development",
       authConfig: {
         clientId,
-        scope: "openid email profile organization imodelhub context-registry-service:read-only product-settings-service urlps-third-party offline_access",
```
Comment (calebmshafer) [inline src/backend/main.ts:66]: without offline_access we won't get a refresh token, right?

**Review by calebmshafer — CHANGES_REQUESTED** [review]

File: src/backend/main.ts (hunk 1 above)
Comment (williamkbentley) [reply src/backend/main.ts:66]: I copied the list of scopes from developer.bentley.com. offline_access isn't in the list. I guess native clients get `offline_access` by default? I'll follow up with that team, but I'll add this scope back.

Comment (calebmshafer) [reply src/backend/main.ts:66]: Ah okay, maybe developer.bentley.com adds it automatically or it's not needed anymore for a client created there.

**Review by BaoTon — APPROVED** [review]
LGTM

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**